snap/snapcraft.yaml

```diff
@@ -27,6 +33,10 @@ apps:
       - removable-media
 
 parts:
+  scripts:
+    plugin: dump
+    source: ./scripts
```

Comment:
What does this do?

Reply:
I'm unable to update LD_LIBRARY_PATH directly in the snap to add the `blas` and the `lapack` folders, so I did a little script that adds the folder where they are, and use it to launch the binary.

Reply:
I tried both with a global `environment` entry, and a local with in the `app`, but in both cases it didn't append worked (and I used ${LD_LIBRARY_PATH:+$LD_LIBRARY_PATH:} to append what I needed).